cesql/spec.md

```diff
@@ -117,7 +117,7 @@ CESQL defines 3 different literal kinds: integer numbers, `true` or `false` bool
 digit ::= [0-9]
 number-literal ::= digit+
 
-boolean-literal ::= "true" | "false"
+boolean-literal ::= "true" | "false" | "TRUE" | "FALSE"
```

Comment:
Is there a format that means "any case", so that `TrUe` would be acceptable too?

Reply:
@MaryamTaj I couldn't find a way... can you add a comment to indicate that it's case insensitive? 
e.g.
```
boolean-literal ::= "true" | "false"       (* Case insensitive *)
```

Reply:
@MaryamTaj we chatted about this on today's call... are you ok with the above suggestion?

Reply:
@duglin Thank you so much for letting me know! I'd be happy to make the changes:)

Reply:
So just to check, this is intended to match True and False (or, less likely, tRUE and fALSE)? I'm pretty sure that's the aim, just checking.